src/main/kotlin/org/medaware/anterogradia/libs/Standard.kt

```diff
@@ -60,20 +86,55 @@ class Standard(val runtime: Runtime) {
         return _else.evaluate(runtime)
     }
 
-    @DiscreteFunction(identifier = "equal", params = ["a", "b"])
+    @DiscreteFunction(identifier = "equal", params = [CMP_LEFT, CMP_RIGHT])
     fun equal(a: Node, b: Node): String = if (a.evaluate(runtime) == b.evaluate(runtime)) "true" else "false"
```

Comment:
does this work if you change ```params```, but don't change the parameters in ```equal()```?

Reply:
Yep! It does! Again, the reflections engine is pretty flexible when it comes to naming things. But again, I'll totally agree with your point of view that it is quite confusing. Thanks!
Resolved in 289b87b.